packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -56,7 +56,7 @@
             >
           </div>
           <div
-            class="font-medium pt-2 text-18px leading-28px"
+            class="font-medium pt-2 text-18px leading-28px truncate"
```

Comment:
I think we need one more tweak to avoid the text running right up against the card border on the left:

```suggestion
            class="font-medium p-2 text-18px leading-28px truncate"
```

Before change:
![no padding](https://user-images.githubusercontent.com/8340719/209677164-6a91e657-f6a0-48cc-91b4-4ffcf245cf9c.png)

After change:
![with padding](https://user-images.githubusercontent.com/8340719/209677172-4415d0cf-1aa3-46f9-9279-0a415cefbe84.png)

After that we can merge :)

Reply:
@marktnoonan I have addressed the PR. Good catch! 👍🏻